docs/open-api/lineage.yaml

```diff
@@ -0,0 +1,281 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+
+openapi: 3.0.2
+info:
+  title: OpenLineage
+  version: 2-0-2
+  description: OpenLineage is an open source **lineage and metadata collection API** for the data ecosystem.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /api/lineage:
+    post:
+      summary: Send an event related to the state of a run
+      description: Updates a run state for a job.
+      operationId: postRunEvent
+      tags:
+        - OpenLineage
+      requestBody:
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: '#/components/schemas/RunEvent'
+                - $ref: '#/components/schemas/DatasetEvent'
+                - $ref: '#/components/schemas/JobEvent'
+      responses:
+        200:
+          description: OK
```

Comment:
Please provide the complete response, including the body and any other exceptional responses.

Reply:
updated